rest-api/src/main/java/gov/cms/qpp/conversion/api/services/AuditServiceImpl.java

```diff
@@ -108,13 +122,13 @@ private boolean noAudit() {
 
 	private Metadata initMetadata(Converter.ConversionReport report, MetadataHelper.Outcome outcome) {
 		Metadata metadata = MetadataHelper.generateMetadata(report.getDecoded(), outcome);
-		metadata.setFileName(report.getFilename());
+		metadata.setFileName(report.getQrdaSource().getName());
 		return metadata;
 	}
 
-	private CompletableFuture<String> storeContent(InputStream content) {
+	private CompletableFuture<String> storeContent(InputStream content, long size) {
```

Comment:
What do you think about changing the contract of this method to accept a Source?

Reply:
I thought against it.  I wanted to keep `Source` cordoned off from the rest of the `rest-api` project as much as possible since `Source` is from the `converter` project.  I'm not aware of which way would be more beneficial though, so I can be convinced otherwise.

Reply:
K, storeContent is only used within this implementation, so it wouldn't require a change to any of the interface's contracts. AuditServiceImpl.java is already dependent on Source so it wouldn't introduce any thing new in trms of dependencies. The thought is that it reduces verbosity of the call to storeContent.
From 
`storeContent(qppSource.toInputStream(), qppSource.getSize())` 
to
`storeContent(qppSource)` x 8 invocations
Not a deal breaker though.